recipes/orca-test/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "orca_test" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/UDST/orca_test/archive/v{{ version }}.tar.gz
+  sha256: 3107b31578b2f6e45c5e57b8bc45c94bf2d59f3d3ad1d7162bf36537f7209cca
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - numpy >=1.0
+    - orca >=1.3.0
+    - pandas >=0.12
+    - pip
+    - python
+  run:
+    - numpy >=1.0
+    - orca >=1.3.0
+    - pandas >=0.12
+    - python
+
+test:
+  imports:
+    - orca_test
+
+about:
+  home: https://github.com/urbansim/orca_test
```

Comment:
Could you clarify why the package is being downloaded from the `UDST` organization, but the `home` is `urbansim`?

Reply:
Will change it to `UDST`, thanks for noticing it!